test/gtest/common/test_helpers.cc

```diff
@@ -127,3 +127,62 @@ message_stream::~message_stream() {
 } // detail
 
 } // ucs
+
+namespace ucp {
+
+const size_t
+data_type_desc_t::_iov_cnt_limit = sizeof(data_type_desc_t::_iov) /
+                                   sizeof(data_type_desc_t::_iov[0]);
+
+data_type_desc_t &
+data_type_desc_t::make(ucp_datatype_t datatype, void *buf, size_t length,
+                       size_t iov_cnt)
+{
+    EXPECT_FALSE(is_valid());
+
+    if (_length == 0) {
```

Comment:
Is contig support added for convenience (for using getter buf and count methods)?
It is not possible to create 0-length contig buffer now

Reply:
Contig is used in stream tests, here 0-length buffer is meaningless, but it can be added for tag in future if needed.